src/main/io/vtx.c

```diff
@@ -63,7 +63,11 @@ void pgResetFn_vtxSettingsConfig(vtxSettingsConfig_t *vtxSettingsConfig)
 #endif
     vtxSettingsConfig->pitModeFreq = VTX_TABLE_DEFAULT_PITMODE_FREQ;
     vtxSettingsConfig->lowPowerDisarm = VTX_LOW_POWER_DISARM_OFF;
-    vtxSettingsConfig->softserialAlt = 0;
+#ifdef AT32F4
```

Comment:
This is my suggestion, but it will cause minor problems - softserial will behave differently on AT32F4. But .. users can reconf accordingly (and IIRC correct SA implementation should accept both versions)